fastlane/helper/plugin_scores_helper.rb

```diff
@@ -0,0 +1,184 @@
+module Fastlane
+  module Helper
+    module PluginScoresHelper
+      require 'faraday'
+
+      class FastlanePluginRating
+        attr_accessor :key
+        attr_accessor :description
+        attr_accessor :value
+
+        def initialize(key: nil, description: nil, value: nil)
+          self.key = key
+          self.description = description
+          self.value = value
+        end
+      end
+
+      class FastlanePluginScore
+        attr_accessor :name
+        attr_accessor :downloads
+        attr_accessor :info
+        attr_accessor :homepage
+        attr_accessor :raw_hash
+
+        attr_accessor :data
+
+        def initialize(hash)
+          if ENV["FASTLANE_PLUGIN_DIRECTORY_GITHUB_USER"].to_s.length == 0 || ENV["GITHUB_API_TOKEN"].to_s.length == 0
```

Comment:
I could, but would add noise to the code. Also this won't happen once the code is set up. I figured it's pretty clear for the "user" of the script to then check the variables.